.changeset/weak-otters-love.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-scaffolder': minor
+'@backstage/plugin-scaffolder-backend': minor
+---
+
+Scaffolder's API `/v1/jobs` will accept `templateName` instead of `template` Entity.
```

Comment:
Good to mention this, but not the most important change. This PR introduces a new requirement to construct and pass an `entityClient` to the scaffolder's `createRouter`. Best mention that here as well as include an example of what needs to be added in the backend.

Reply:
Check Check